### PR TITLE
DOCS-3945 Browser and Browser Premium Sampling Configuration Guide

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds a guide about sampling configuration for Browser and Browser Premium sessions.

### Motivation

DOCS-3945

### Additional Notes

Remember to also update the `pull_config_preview.yaml` file before merging!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -459,14 +459,14 @@
...
-      branch: main
+      branch: alai97/browser-monitoring-initialization-snippet-update
       globs:
       - 'packages/rum/README.md'
       options:
         dest_path: '/real_user_monitoring/browser/'
         file_name: '_index.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/browser-sdk/blob/main/packages/rum/README.md"]
+          dependencies: ["https://github.com/DataDog/browser-sdk/blob/alai97/browser-monitoring-initialization-snippet-update/packages/rum/README.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:469]:
```suggestion
          dependencies: ["https://github.com/DataDog/browser-sdk/blob/main/packages/rum/README.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -459,14 +459,14 @@
   - repo_name: browser-sdk
     contents:
     - action: pull-and-push-file
-      branch: main
+      branch: alai97/browser-monitoring-initialization-snippet-update
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:462]:
```suggestion
      branch: main
```

**Review by maycmlee — APPROVED** [review]
Hey @alai97! Going to approve, but was just wondering if the `applicationID` and `clientToken` in the video should be redacted?

File: content/en/real_user_monitoring/guide/sampling-browser-and-browser-premium.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+For more information about tagging and exploring attributes, see [Browser Monitoring][3].
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /real_user_monitoring/browser#setup
+[2]: https://www.datadoghq.com/pricing/?product=real-user-monitoring--session-replay#real-user-monitoring--session-replay
+[3]: /real_user_monitoring/browser#browser-and-browser-premium-sampling-configuration
```
Comment (alai97) [inline content/en/real_user_monitoring/guide/sampling-browser-and-browser-premium.md:133]:
```suggestion
[3]: /real_user_monitoring/browser#tagging
```

**Pull request merged**